src/braket/aws/aws_quantum_task.py

```diff
@@ -419,7 +419,7 @@ def _get_device_execution_windows(self) -> List[DeviceExecutionWindow]:
             if not device_arn:
                 device_arn = self.metadata(use_cached_value=False).get("deviceArn")
             device_capabilities = json.loads(
-                self._aws_session.get_device(device_arn)["deviceCapabilities"]
+                self._aws_session.get_device(device_arn, True)["deviceCapabilities"]
```

Comment:
```suggestion
                self._aws_session.get_device(device_arn, use_cached_value=True)["deviceCapabilities"]
```
I'd make it explicit what the keyword does.

Reply:
Good point.